2021-01-18_Liu_reactions.pbtxt

```diff
@@ -0,0 +1,1480 @@
+      measurements {
+        analysis_key: "isolated_weight"
+        type: YIELD
+        percentage {
+          value: 93.5
+          precision: 0.5
+        }
+      }
+      measurements {
+        analysis_key: "quantitative NMR"
+        type: PURITY
+        percentage {
+          value: 99.0
+        }
+      }
+      measurements {
+        analysis_key: "isolated_weight"
+        type: AMOUNT
+        amount {
+          mass {
+            value: 27.75
+            units: GRAM
+          }
+        }
+      }
+      isolated_color: "white"
+      texture {
+        type: POWDER
+      }
+      reaction_role: PRODUCT
+    }
+    analyses {
+      key: "13C NMR"
+      value {
+        type: NMR_13C
+        details: "101 MHz, CDCl3"
+        is_of_isolated_species: true
+        data {
+          key: "peaks"
+          value {
+            string_value: "27.1, 38.4, 62.4, 127.7, 128.3, 129.6, 136.2, 176.3"
+            description: "List of peaks"
+          }
+        }
+      }
+    }
+    analyses {
+      key: "1H NMR"
+      value {
+        type: NMR_1H
+        details: "400 MHz, CDCl3"
+        is_of_isolated_species: true
+        data {
+          key: "peaks"
+          value {
+            string_value: "0.92 (s, 9H), 4.06 (s, 4H), 7.23 - 7.34 (m, 6H), 7.40 (d, J = 7.1 Hz, 4H)"
+            description: "List of peaks"
+          }
+        }
+      }
+    }
+    analyses {
+      key: "HRMS"
+      value {
+        type: HRMS
+        details: "ESI-TOF"
+        is_of_isolated_species: true
+        data {
+          key: "expected"
+          value {
+            float_value: 298.1802
+            description: "Expected m/z"
+          }
+        }
+        data {
+          key: "found"
+          value {
+            float_value: 298.1794
+            description: "Observed m/z"
+          }
+        }
+      }
+    }
+    analyses {
+      key: "isolated_weight"
+      value {
+        type: WEIGHT
+        details: "27.5-28.0 g recovered after workup"
+        is_of_isolated_species: true
+      }
+    }
+    analyses {
+      key: "quantitative NMR"
+      value {
+        type: NMR_1H
+        details: "Quantitative NMR using 1,1,2,2-tetrachloroethane (>98%, purchased from Alfa Aesar) in CDCl3"
+        is_of_isolated_species: true
+      }
+    }
+    analyses {
+      key: "thin film IR"
+      value {
+        type: IR
+        details: "neat film, NaCl"
+        is_of_isolated_species: true
+        data {
+          key: "peaks"
+          value {
+            string_value: "3064, 3031, 2973, 2932, 2906, 2872, 1751, 1496, 1479, 1456, 1273, 1116, 1029, 738, 698"
+            description: "List of peaks [cm-1]"
+          }
+        }
+      }
+    }
+  }
+  provenance {
+    city: "Cambridge, MA"
+    doi: "10.15227/orgsyn.095.0080"
+    publication_url: "http://www.orgsyn.org/demo.aspx?prep=v95p0080"
+    record_created {
+      time {
+        value: "01/18/2021, 10:02:57"
+      }
+      person {
```

Comment:
You should update this to include yourself! If you'd like to keep both of us, then you can add a "RecordModified" event